
From: Adrian Bunk <bunk@fs.tum.de>

  CC      drivers/media/radio/radio-maestro.o
drivers/media/radio/radio-maestro.c: In function `maestro_radio_init':
drivers/media/radio/radio-maestro.c:273: sorry, unimplemented: inlining 
failed in call to 'radio_install': function body not available
drivers/media/radio/radio-maestro.c:291: sorry, unimplemented: called from here
drivers/media/radio/radio-maestro.c:273: sorry, unimplemented: inlining 
failed in call to 'radio_install': function body not available
drivers/media/radio/radio-maestro.c:295: sorry, unimplemented: called from here
make[3]: *** [drivers/media/radio/radio-maestro.o] Error 1

Signed-off-by: Adrian Bunk <bunk@fs.tum.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/media/radio/radio-maestro.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN drivers/media/radio/radio-maestro.c~radio-maestroc-remove-an-inline-fwd drivers/media/radio/radio-maestro.c
--- 25/drivers/media/radio/radio-maestro.c~radio-maestroc-remove-an-inline-fwd	Thu Jul 29 15:01:47 2004
+++ 25-akpm/drivers/media/radio/radio-maestro.c	Thu Jul 29 15:01:47 2004
@@ -270,7 +270,7 @@ static int radio_ioctl(struct inode *ino
 	return ret;
 }
 
-inline static __u16 radio_install(struct pci_dev *pcidev);
+static __u16 radio_install(struct pci_dev *pcidev);
 
 MODULE_AUTHOR("Adam Tlalka, atlka@pg.gda.pl");
 MODULE_DESCRIPTION("Radio driver for the Maestro PCI sound card radio.");
@@ -324,7 +324,7 @@ inline static __u16 radio_power_on(struc
 	return (ofreq == radio_bits_get(dev));
 }
 
-inline static __u16 radio_install(struct pci_dev *pcidev)
+static __u16 radio_install(struct pci_dev *pcidev)
 {
 	if(((pcidev->class >> 8) & 0xffff) != PCI_CLASS_MULTIMEDIA_AUDIO)
 		return 0;
_
